website/docs/ParameterizeTests.md

```diff
@@ -0,0 +1,167 @@
+---
+id: parameterize-tests
+title: Parameterize Tests
+---
+
+You can simply parameterize tests on a test level, e.g.:
```

Comment:
I sometimes dynamically create tests like this:
```js
function testComponent(componentName, options) {
  it(`should test my ${componentName}`, async () => {
    await browser.url(`/${componentName}`);
    await expect($('input')).toHaveValue(options.expectedValue);
  });
}
```
Do you think it would be good to add how you can dynamically create these tests?

Reply:
Totally!